fun will_throw
{}
var exc = void
try {[ will_throw()... ]
}
catch const e {
 e
fun will_throw
{
 void
try {
 [!will_throw... ]
}
catch const e {
  exc =delete e.val
exc = e.v
} yield 55
}
[ will_throw()... ]
assert exc}assert exc
